tests/testthat/test-dataset_vignette.R

```diff
@@ -1,23 +1,34 @@
 library(admiral.test)
 
-## Test 1: a 'knitr_kable' object is output when run outside pkgdown ----
-test_that("dataset_vignette Test 1: a 'knitr_kable' object is output when run outside pkgdown", {
+# dataset_vignette
+## Test 1: A 'knitr_kable' object is output when run outside pkgdown ----
+test_that("Test 1: A 'knitr_kable' object is output when run outside pkgdown", {
   expect_s3_class(dataset_vignette(head(admiral_dm)), "knitr_kable")
 })
 
-## Test 2: a 'datatables' object is output when run inside pkgdown ----
-test_that("dataset_vignette Test 2: a 'datatables' object is output when run inside pkgdown", {
+## Test 2: A 'datatables' object is output when run inside pkgdown ----
+test_that("Test 2: A 'datatables' object is output when run inside pkgdown", {
   Sys.setenv(IN_PKGDOWN = "true")
   on.exit(Sys.setenv(IN_PKGDOWN = ""))
   expect_s3_class(dataset_vignette(head(admiral_dm)), "datatables")
```

Comment:
Can you also update the calls like this for test data?  Or just make some simple dummy data and remove the dependency all together.

```suggestion
  expect_s3_class(dataset_vignette(head(admiral.test::admiral_dm)), "datatables")
```